packages/driver/cypress/integration/commands/waiting_spec.js

```diff
@@ -68,7 +68,7 @@ describe('src/cy/commands/waiting', () => {
 
           return null
         })
-        .wait('@fetch').then((xhr) => {
+        .wait('@fetch.response').then((xhr) => {
```

Comment:
why is `.response` now needed over just the alias? Is this something that will break users?